tbx/project_styleguide/templates/patterns/pages/team/team_detail.html

```diff
@@ -21,8 +21,8 @@
         </div>
 
         {% if page.intro %}
-            <div class="grid__intro streamfield__intro">
-                <div class="heading heading--four">{{page.intro|richtext}}</div>
+            <div class="grid__intro streamfield__intro rich-text">
```

Comment:
Suggestion: Could this and the other instances in `work_page.html`, `service_page.html` use an include to the `intro_block.html`? E.g.

```
{% include "patterns/molecules/streamfield/blocks/intro_block.html" with value=page.intro %}
```

Reply:
ah interesting - will have a think.

Reply:
I've created a new common intro file at `molecules/intro/intro.html` - this is then included by all other places where we use an intro, including the intro streamfield block. Think that's slightly cleaner than including the block template directly.